app/views/orders/_cart_table.html.erb

```diff
@@ -0,0 +1,63 @@
+<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet">
```

Comment:
use `provide_for` or `content_for` to load `script` or `link` on head HTML
Suggest: use `yield`